src/Modal/ScreenCover.js

```diff
@@ -0,0 +1,23 @@
+import React, { Fragment } from 'react'
```

Comment:
The screen cover would be handy to use for `CircularProgress` as well, so it could be pulled out to its own component.

Reply:
Good catch!